docs/mouser/about.md

```diff
@@ -0,0 +1,56 @@
+---
+id: about
+title: Mouser
+custom_edit_url: null
+---
+
+## Overview
+
+![Alt](100x100.png) Created mainly for SLU researchers, this software is used for tracking the data of animal experiments. Running as a desktop app, users of the software can create and manage experiments in one easy place. Animals can be scanned into the system using RFID, where they can be quickly accessed for data collection and grouping. Over the course of an experiment, the user is able to take measurements of the animals using devices connected to the computer.
```

Comment:
It would be good to emphasizes the value of this software. Here is an example of commercial software with what looks like overlapping functionality: https://mosaicvivarium.net. You could borrow some ideas from there.